.env.oss

```diff
@@ -39,6 +39,7 @@ S3_BUCKET=
 SAILTHRU_KEY=
 SAILTHRU_SECRET=
 SEGMENT_WRITE_KEY=
+SEGMENT_WRITE_KEY_SERVER=
```

Comment:
I'm not sure why we have all these non-OSS placeholders in `.env.oss`, seems odd.

Reply:
I believe this was just a result of the desktop + mobile consolidations- getting the two environments together.

Reply:
Listing all env variables being used by a program is just _a_ way to document those. After cloning the repo and copying this file to `.env` it should be clear to the dev what configuration needs to be done.

Reply:
I get that, I guess I'm just musing if there's a better way. For instance @izakp and I briefly discussed something similar with regards to Metaphysics secrets, plus https://github.com/artsy/post_mortems/issues/28 (caused by leaky `.env` file), so I figured I'd raise a PR comment/question.

Agreed that its a pattern we follow, who knows? Maybe that'll change someday...

Reply:
Oh I definitely think the consolidated config file is a better approach.